src/api_methods.py

```diff
@@ -1,159 +1,176 @@
-import requests
-import getpass
 import json
-
-import pandas as pd
+import getpass
 import urllib.request
+
 import tqdm
+import requests
+import pandas as pd
+import pathlib
+
+from api_endpoints import url_account, url_info_client, url_login, url_positions
+
+
+def get_config(fname):
+    """Read credentials from config file.
+
+    Parameters
+    ----------
+    fname : str
+        Complete path to config file.
 
-def get_session_id(username = None, password = None):
+    Returns
+    -------
+    config : config
     """
-    Get sessionId for a username and password.
-    
+    import configparser
+    config = configparser.ConfigParser()
+    _fname = pathlib.Path(fname).expanduser().absolute()
+    config.read(_fname)
+    return config
+
+
+def get_session_id(username=None, password=None, config=False):
+    """Get sessionId for a username and password.
+
     Parameters
     ----------
     username: str
-    
     password: str
-    
+
     Returns
     -------
     str
     """
     sess = requests.Session()
 
     # Prepare payload
-    _payload = {'isPassCodeReset': False,
-                'isRedirectToMobile': False}
-    
-    _need_username = username is None
-    _need_password = password is None
-    
+    _payload = {"isPassCodeReset": False, "isRedirectToMobile": False}
+
+    _need_username = username is None and not config
+    _need_password = password is None and not config
+
+    if config:
+        config = get_config(config)
+        username = config["LOGIN"]["username"]
+        password = config["LOGIN"]["password"]
+
     if _need_username or _need_password:
-        
-        _payload['username'] = input('Input username')
-        _payload['password'] = getpass.getpass()
-        
+
+        _payload["username"] = input("Username: ")
+        _payload["password"] = getpass.getpass()
+
     else:
-        
-        _payload['username'] = username
-        _payload['password'] = password
-        
+
+        _payload["username"] = username
+        _payload["password"] = password
+
     _payload = json.dumps(_payload)
 
-    # Prepare header
-    _header = {'content-type': 'application/json'}
+    #  Prepare header
+    _header = {"content-type": "application/json"}
 
     # Request access
-    _response = sess.post(url     = url_login, 
-                          headers = _header, 
-                          data    = _payload)
-    
+    _response = sess.post(url=url_login, headers=_header, data=_payload)
+
     if _response.ok != True:
         print(_response.text)
-        raise SystemExit('Unable to retrive intAccount value.')
-    
+        raise SystemExit("Unable to retrive intAccount value.")
+
     sess.close()
 
-    return _response.headers['Set-Cookie'].split(';')[0].split('=')[-1]
+    return _response.headers["Set-Cookie"].split(";")[0].split("=")[-1]
+
+
+def get_int_account(session_id=None):
+    """Get intAccount values for a sessionId value.
```

Comment:
I start the docstring after the `"""` because visually it helps me read quickly. 

Is it that bad?